AI/sccmain.py

```
@@ -0,0 +1,142 @@
+
+# ------------------- 영상 스트리밍 -------------------
+def generate_frames_game1(target_width=1080, target_height=607):  # 16:9
+    mp_drawing = mp.solutions.drawing_utils
+    mp_holistic = inference.mp_holistic
+
+    while True:
+        frame = camera.get_frame()
+        if frame is None:
+            continue
+
+        predicted_action, confidence, game_result = inference.process_frame(frame)
```

Ruff (0.13.1)

Unpacked variable `predicted_action` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)